specification/ai/data-plane/OpenAI.Assistants/OpenApiV3/2024-02-15-preview/assistants_generated.yaml

```diff
@@ -340,7 +340,7 @@ paths:
               required:
                 - file
                 - purpose
-  /files/{fileId}:
+  /files/{fileId}/{file_id}:
```

Comment:
it seems wrong, `{fileId}/{file_id}`.

Reply:
That's odd! I just ran the compile command. I will manually remove :)